src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotSuitableSDOCallbackHandler.java

```diff
@@ -60,6 +63,18 @@ private CallbackResponse submitNotSuitableSDO(CallbackParams callbackParams) {
             .build();
     }
 
+    private CallbackResponse validateNotSuitableReason(CallbackParams callbackParams) {
+        List<String> errors = new ArrayList<>();
+        var reason = callbackParams.getRequest().getCaseDetails().getData().get("reasonNotSuitableSDO");
+        if (reason.toString().length() > 158) { // set at 158, because the method adds {input=} characters
```

Comment:
you can add a constant with value 150 and then use reason.toString().length()>CONSTANT+"input=".length
that moves the "magic number" to an easily located place (or even a configurable one if you make it a field and put the value in application.yml) and the code includes why using 158 instead of 150. That constant can also be used in the error message